base_custom_info/models/custom_info.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# © 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class CustomInfoTemplate(models.Model):
+    """Defines custom properties expected for a given database object."""
+    _name = "custom.info.template"
+    _description = "Custom information template"
+    _sql_constraints = [
+        ("name_model",
+         "UNIQUE (name, model_id)",
+         "Another template with that name exists for that model."),
+    ]
+
+    name = fields.Char(translate=True)
+    model_id = fields.Many2one(comodel_name='ir.model', string='Model')
```

Comment:
Is it required a hard link? Isn't enough with a char with the name of the model? This will ease the data inclusion.